modules/calypsoify/mods-gutenberg.js

```diff
@@ -1,7 +1,7 @@
 /* global calypsoifyGutenberg */
 jQuery( function( $ ) {
-	var editPostHeaderInception = setInterval( function() {
-		var $editPostHeader = $( '.edit-post-header' );
+	const editPostHeaderInception = setInterval( function() {
```

Comment:
Let's actually keep it `var` - since our configs is not yet set up to approve these, it's throwing errors on build and in the CI.

Reply:
Swapping back, but our precommit checks are conflicting with some of our configs in ci it seems --

![screen shot 2018-10-30 at 10 28 37 am](https://user-images.githubusercontent.com/941023/47725280-a6cf2600-dc2e-11e8-885f-f50e6acdd0a5.png)

Reply:
yeah, that's annoying and we should fix that inconsistency. For now, you can add `/* eslint-disable no-var */` to the files.

Reply:
That's a known issue, you can use `--no-verify` here, Travis does not run the same tests.

More details about this:
p1540478539000100-slack-jetpack-crew